l3kernel/l3luatex.dtx

```diff
@@ -554,6 +616,61 @@ end
 %    \end{macrocode}
 % \end{macro}
 %
+% \begin{macro}{ltx.utils.require}
+%   Loads a Lua module. This function loads the module just like the standard
+%   Lua global function |require|, with two key differences:
+%   \begin{enumerate}
+%     \item This function is guaranteed not to throw an error.
```

Comment:
It's not though. If the module is found, but loading it triggers an error, then `require` will throw an error. (I think that's correct behavior, but it contradicts this statement)